### DISCUSSION TITLE
Set Output Format Manually

### DISCUSSION THREAD

**Discussion by codegrizzl** [cwilby/nextcloud-workflow-media-converter#457] [General]
I would like to set a output format which is currently not listed.

Is there a possibility to set a format in a different way. As without the conversion job does not seem to start.

If this is not possible, it would be nice to be able to define different output formats in settings or set them manually as well.

This can also be helpfull if admin wants to restrict certain formats. And will make this flow more versatile.

Regards
Patrick

**Comment by cwilby** [comment]
I think it makes sense to trade off and add a "Custom" or "Other" option.  Will look into it next.

**Comment by Mr-Kanister** [comment]
Hi,
I'd like to revisit this topic as I'd like to be able to convert images to `.avif`. On the other hand I believe it is impractical to add a dropdown option for each and every file extension. So a custom option would be indeed very helpful here.

Best regards
